Woodog/templates/partials/base.html

```diff
@@ -118,7 +118,7 @@
 
 <nav id="header" class="navbar navbar-expand-lg">
   <div class="container-fluid">
-    <a class="navbar-brand logo" href="#"><img src="{% static 'img/woodog-logo.svg' %}"></a>
+    <a class="navbar-brand logo" href="#"><img src="{% static 'img/woodog-logo1.png' %}"></a>
```

Comment:
Hi @ishitakapoor26, Thanks for your contribution.
Can you please remove the background of the logo? It'll be good if we just have the icon and the text 'Woodog'.

Reply:
![image](https://user-images.githubusercontent.com/58631762/122527223-c2360c00-d038-11eb-821a-3e3018729172.png)

Here's the screenshot,

Reply:
Also, Add `style` attribute for `<img>` to give width.

```html
<img src="/static/img/woodog-logo1.png" style="width: 80px;">
```